easybuild/easyconfigs/c/CPLEX/CPLEX-22.10-GCCcore-11.2.0-Python-3.9.6.eb

```diff
@@ -0,0 +1,26 @@
+name = 'CPLEX'
+version = '22.10'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
drop the versionsuffix, we made python 3 the default from the 2020a toolchain generation onwards. The filename should be updated to reflect this as well
```suggestion
```